wls-gui-wahllokalsystem/src/types/User.ts

```diff
@@ -1,34 +1,28 @@
 import type { WahlbezirksArtEnum } from "@/types/wahlbezirksArtEnum.ts";
 import type { WahlMetaData } from "@/types/wlsTypes/WahlMetaData.ts";
 
-class User {
-  username = "";
-  email = "";
-  userEnabled? = false;
-  wahltagID? = "";
-  wahltag? = "";
-  wahlbezirkID? = "";
-  wahlbezirkNummer? = "";
-  wahlbezirksArt?: WahlbezirksArtEnum;
-  pin = "";
-  authorities = new Set<string>();
-  wahlMetaData?: WahlMetaData[];
+export interface User {
+  username: string;
+  email: string;
+  userEnabled?: boolean | undefined;
+  wahltagID?: string | undefined;
+  wahltag?: string | undefined;
+  wahlbezirkID?: string | undefined;
+  wahlbezirkNummer?: string | undefined;
+  wahlbezirksArt?: WahlbezirksArtEnum | undefined;
+  pin: string;
+  authorities: Set<string>;
+  wahlMetaData?: WahlMetaData[] | undefined;
 }
```

Comment:
❗ bei `createUserLocalDevelopment` müssen dann entsprechend die Props noch ergänzt werden


```suggestion
export interface User {
  username: string;
  email: string;
  userEnabled: boolean | undefined;
  wahltagID: string | undefined;
  wahltag: string | undefined;
  wahlbezirkID: string | undefined;
  wahlbezirkNummer: string | undefined;
  wahlbezirksArt: WahlbezirksArtEnum | undefined;
  pin: string;
  authorities: Set<string>;
  wahlMetaData: WahlMetaData[] | undefined;
}
```

Reply:
fixed [60d09cb](https://github.com/it-at-m/Wahllokalsystem/pull/1310/commits/60d09cb22efa9325bfd08848136e641e509c18eb)